packages/react-ui/src/widgets/ComparativeCategoryWidgetUI.js

```diff
@@ -0,0 +1,696 @@
+import {
+  Box,
+  Button,
+  Checkbox,
+  darken,
+  Divider,
+  InputAdornment,
+  lighten,
+  Link,
+  makeStyles,
+  SvgIcon,
+  TextField,
+  Tooltip,
+  Typography,
+  useTheme,
+  withStyles
+} from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
+import React, { useMemo, useState } from 'react';
+import PropTypes from 'prop-types';
+import AnimatedNumber, {
+  animationOptionsPropTypes
+} from '../custom-components/AnimatedNumber';
+
+/**
+ * Enum for ComparativeCategoryWidgetUI order types. 'RANKING' orders the data by value and 'FIXED' keep the order present in the original data
+ * @enum {string}
+ */
+export const ORDER_TYPES = {
```

Comment:
Can't we share same order types from CategoryWidget?

Reply:
Sure, we could move the `export const ORDER_TYPES` to `CategoryWidgetUI` and use that instead of writing them to `CategoryWidgetUI`, so we can import them separately from `ComparativeCategoryWidgetUI`.

Reply:
Solved